src/Management/Commands/BanCommand.cs

```diff
@@ -0,0 +1,82 @@
+﻿using sodoffmmo.Attributes;
+using sodoffmmo.Core;
+using System.Numerics;
+
+namespace sodoffmmo.Management.Commands;
+
+[ManagementCommand("ban", Role.Moderator)]
+class BanCommand : IManagementCommand {
+    public void Handle(Client client, string[] arguments) {
+        if (arguments.Length < 1) {
+            client.Send(Utils.BuildServerSideMessage("Ban: Invalid number of arguments", "Server"));
+            return;
+        }
+        string name = string.Join(' ', arguments);
+        Client? target = client.Room.Clients.FirstOrDefault(x => x.PlayerData.DiplayName == name); // Find in this room.
+        if (target == null) { // Find through all the rooms.
+            foreach (Room room in Room.AllRooms()) {
```

Comment:
This can be considered expensive. Not mentioning that now we have as many rooms as banned players